Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -437,6 +437,28 @@ def condition_(self, fragment, stepList, key, hasHarvest):
     '--procModifiers': 'trackingMkFitDevel'
 }
 
+# mkFit for phase-2 initialStep tracking
+class UpgradeWorkflow_trackingMkFitPhase2(UpgradeWorkflowTracking):
```

Comment:
Why a different class (and then a different offset)? I don't see anything that could not have been done with `UpgradeWorkflow_trackingMkFit` as is using the same `*.7` offset. Or am I missing something?

Reply:
(Ok, not with the same suffix but within the same class yes)

Reply:
Phase-2 would need a different process modifier, and should not modify step2 on the other hand.
That's why we went with a different class, because it was the simplest solution. We may review this (and remove the class) when we'l have more than one iteration working in phase-2, so that we can reduce otherwise-needed modifications to the mkFit procModifiers?

Reply:
Yea, this was clear. And yes, most probably it's easier to have this rather than "disabling" the other iterations with something like `~phase2_tracker`. Maybe I would have inverted the logic here (so a `UpgradeWorkflow_trackingMkFitInitialStep`) just to have it clear what's happening in the wf from the name. Anyway as you say, this will need to be reviewed anyway later.